db/migrate/20220429031113_create_canonical_armors.rb

```diff
@@ -4,11 +4,17 @@ class CreateCanonicalArmors < ActiveRecord::Migration[6.1]
   def change
     create_table :canonical_armors do |t|
       t.string :name, null: false
+      t.string :item_code, null: false, unique: true
       t.string :weight, null: false
       t.string :body_slot, null: false
-      t.decimal :unit_weight, precision: 5, scale: 1
+      t.string :magical_effects
+      t.decimal :unit_weight, precision: 5, scale: 2
       t.boolean :dragon_priest_mask, default: false
       t.boolean :quest_item, default: false
+      t.boolean :unique_item, default: false
+      t.boolean :enchantable, default: true
```

Comment:
Not all non-magical items can be enchanted. This should always be set to false for models that are canonically enchanted but is really intended to determine whether non-enchanted items can _be_ enchanted.